### ISSUE TITLE
Slaanes - Selecting your General 2

### ISSUE THREAD

**Issue by JozkoKanec** [BSData/warhammer-age-of-sigmar#3269]
Describe the bug
- Including some leaders (i.e. Dexcessa, Infernal Enrapturess...) returns error "cannot have any selections of General (hidden)"

To Reproduce

Go to 'Age of Sigmer - Slaanesh'
Click on 'Leaders +'
add Dexcessa to your list
See error
Expected behavior
Should not return error

Screenshots
![image](https://github.com/BSData/warhammer-age-of-sigmar/assets/147946975/1312c1db-7470-4bc1-99d2-730e4d435583)

![image](https://github.com/BSData/warhammer-age-of-sigmar/assets/147946975/fea8929b-5dc5-406a-9df4-2b48af00646f)

Device and data (please complete the following information):

OS: Android
App being used to create lists: Battlescribe
App version: 2.03.27.341
Data release version: 3.3.23

**Comment by JozkoKanec** [comment]
Expected behavior
Should not return THIS error.
No error after dexcessa is selected to be General

**Comment by exaegrum** [comment]
Hm. I can't reproduce this one. Is there anything else selected besides the 3 units of battleline?

**Comment by exaegrum** [comment]
Ah. Do you have the Invaders host selected? The mobile application's general selection appears to be having some sort of conflict with all their HEROES being able to count as generals. I'll see if I can figure out why.

**Comment by JozkoKanec** [comment]
Yes, correct. I selected invaders at random (just to reduce error count) and switching to godseekers removed the error
